src/sass/components/ConfirmationModal.scss

```diff
@@ -0,0 +1,35 @@
+.confirmation-modal, .confirmation-blurred-blocker {
+  position: absolute;
+  top:0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 2;
+}
+
+.confirmation-modal {
+  width: 30%;
+  min-width: 320px;
+  height: 200px;
+  margin: auto;
+  position: fixed;
+  z-index:3;
```

Comment:
If you bump `z-index` to at least `9001` the nav stuff will be under the overlay too. The `.usa-nav` is USWDS has a `z-index` of `9000` which is why it's not.